.github/workflows/deploy-app.yaml

```diff
@@ -6,6 +6,8 @@ env:
   DEPLOY_PATH: '/home/w3reg/app.w3reg.org'
   BACKUP_PATH: '/home/w3reg/backups'
   STAGING_PATH: '/home/w3reg/staging.w3reg.org'
+  VITE_APP_AVAILABLE_CHAINS: 'paseo_people'
+  VITE_APP_DEFAULT_CHAIN: 'paseo_people'
```

Comment:
what is this trying to do?

so development branch is for development.
staging branch is for production build.
deploy app we just copy existing staging application into production using rsync.

Reply:
ok, ut's undone